src/NLog/Targets/IJsonSerializer.cs

```diff
@@ -49,4 +49,15 @@ public interface IJsonSerializer
         /// <returns>Serialized value.</returns>
         string SerializeObject(object value);
     }
+
+    internal interface IJsonSerializerV2
```

Comment:
Makes sense to have it public, as it would allow others to optimize for StringBuilder. Not sure about the naming-convention though, by making it internal, then I could just choose a "random" name :).